packages/flutter_localizations/test/widgets_test.dart

```diff
@@ -581,7 +581,8 @@ void main() {
       )
     );
 
-    // Initial WidgetTester locale is new Locale('', '')
+    // Initial WidgetTester locale is null due to no platform intitializing it.
+    await tester.binding.setLocale('', '');
```

Comment:
What happens if we don't do this?  Can we adjust the expectations below?  Namely, it seems like `$locale` shouldn't return "_", so we should probably adjust the `toString()` implementation of `Locale`

Reply:
Without this, the null handling (after the engine-side change) resolves the locale to the first supported locale, which is en_US. I'll go ahead and check for en_US and roll the engine with this test change.

Reply:
There should be no more Locale("", "") anymore, at least not produced by flutter.